app/src/main/res/values/strings.xml

```diff
@@ -5,6 +5,21 @@
     <string name="menu_discover_title">Entdecken</string>
     <string name="menu_search_title">Suche</string>
 
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+
+    <string name="lorem_ipsum">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.  </string>
+
+
+    <string name="search_button">Suche</string>
```

Comment:
Die deutschen Strings gehören in eine entsprechende deutsche Strings.xml - File. Merge am besten den aktuellen Dev mal wieder hier rein, da sollten inzwischen beide drin sein.

Reply:
nach dem Umbau in den Refactor-Search-Fragment-branch sind nur 2 Strings übrig geblieben. Die beiden habe ich übersetzt, die restlichen gelöscht